static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -172,6 +175,19 @@ function renderInviteMembersButton({
     >
       {action}
     </Hovercard>
+  ) : disabledBySso ? (
+    <Hovercard
+      body={
+        <Fragment>
+          {t(
+            `Your organization must use its single sign-on provider to invite
+            new members.`
```

Comment:
Is the phrasing here right? Are they "inviting" members via SSO, or are they "registering" members?

```suggestion
            `Your organization must use its single sign-on provider to register new members.`
```